docs/ref/models/instances.txt

```diff
@@ -355,10 +355,17 @@ To save an object back to the database, call ``save()``:
 
 .. method:: Model.save(force_insert=False, force_update=False, using=DEFAULT_DB_ALIAS, update_fields=None)
 
+You can view the following sections for a more detailed information each
+argument to this method.
+
+    - :ref:`force_insert <ref-models-force-insert>`.
+    - :ref:`force_update <ref-models-force-insert>`.
+    - :ref:`update-fields <ref-models-update-fields>`.
```

Comment:
This is just personal preference, but I would use a sentence rather than bullet points. I would also combine `force_insert` and `force_update` into a single link and do this in a way so the section titles are visible in the sentence (it will help readability).

Lastly, `update-fields` -> `update_fields`.

Reply:
i have done what you asked for, but i think adding the section titles would likely require more maintenance. It would need updating alongside the titles, if in case they change(although this may be highly unlikely).

Reply:
> It would need updating alongside the titles, if in case they change

Sphinx will insert the title for you if you let it (by not specifying your own).

Reply:
as far as i am aware, it will only update the links. would it update the anchor text as mentioned as of now?

Reply:
Yes, that's what I was telling you.

Reply:
I removed unnecessary titles.